liferay-gradle/blade.servicebuilder.web/build.gradle

```diff
@@ -4,4 +4,8 @@ dependencies {
 	compile 'javax.servlet:javax.servlet-api:3.0.1'
 	compile 'org.osgi:org.osgi.service.component.annotations:1.3.0'
 	compile project(':blade.servicebuilder.api')
+}
+
+testIntegration {
```

Comment:
I think it doesnt run because its not a "workspace" aka the testableTomcat tasks don't exist here.